app/serializers/tracking_serializer.rb

```diff
@@ -5,6 +5,13 @@ def id
 
   link(:self) { rx_v1_prescription_trackings_url(object.prescription_id) }
   link(:prescription) { rx_v1_prescription_url(object.prescription_id) }
+  link(:tracking_url) do
+    if object.delivery_service.casecmp("UPS")
+      "https://wwwapps.ups.com/WebTracking/track?track=yes&trackNums=#{object.tracking_number}"
+    else
```

Comment:
It should only be UPS and USPS, but I'm worried about funky edge cases or wrong data being returned to us somehow. Can we have an explicit `else if` for USPS and then the final `else` that catches everything else be like a blank link or something?